make/debug

```diff
@@ -0,0 +1,7 @@
+DEBUG ?= 0
+ifeq ($(DEBUG), 1)
+	CXXFLAGS:=$(filter-out -O${O},$(CXXFLAGS))
```

Comment:
Would you mind being consistent with formatting? It'd be nice if it was something like:
```
CXXFLAGS := $(filter-out -O${O}, $(CXXFLAGS))
```
(spaces around the operator and the space after the comma).